babel.config.js

```diff
@@ -11,11 +11,25 @@ function resolvePath(sourcePath, currentFile, opts) {
 
 let defaultPresets;
 
-// We release a ES version of Material-UI.
-// It's something that matches the latest official supported features of JavaScript.
-// Nothing more (stage-1, etc), nothing less (require, etc).
+// evergreen build
+// see https://next.material-ui.com/guides/minimizing-bundle-size/#evergreen-build
 if (process.env.BABEL_ENV === 'es') {
-  defaultPresets = [];
+  defaultPresets = [
+    [
+      '@babel/preset-env',
+      {
+        modules: false,
+        // snapshot of the latest version of evergreen browsers
+        targets: {
+          chrome: 73,
+          edge: 18,
+          firefox: 66,
+          node: 8,
```

Comment:
?

```suggestion
          node: 12,
```

Reply:
I'll have to check how LTS works in node but I would expect new features to be backported. If this is the case we can stick with node 8. Maybe we need to tell browserslist that this means the latest node 8 version.

Reply:
I think *latest active node version* makes sense here (opposed to *maintenance node version* in the prod bundle). This would mean 12.2.

Reply:
I don't follow, does /es targets evergreen platforms? Why should we consider what's backported and what's not? The LTS is v10.

Reply:
Multiple major node releases are in LTS mode. Node 8 is in maintenance which means since 2019 no more features but bug fixes. Node 10 is in active LTS. Node 12 is current and is the next active LTS candidate. 

The evergreen term doesn't make much sense for node since node is not updated automatically. You still have to actively upgrade it.

We can go with *latest active LTS node version* though I'm confused why you propose node 12 and after I accept it you want to support node 10.

Reply:
I don't want to support node 10 over 12. In this case, I think that we should focus on the "last 1 version". Few people use the /es output. It was introduced for people who want to access our sources.

Reply:
> Node 8 is in maintenance which means since 2019 no more features but bug fixes. 

This was just an assumption. I can't find any documentation about the difference between *maintenance* and *active*. It seems like the only difference is update frequency. So *latest node maintenance LTS* should cover the same syntax as *latest node active LTS* though they might be out-of-sync for a couple of weeks.

Reply:
I'm not sure what the problem is here. I'm clearly improving the current situation for the /es build since. Low usage number might be due to incorrect documentation and little to no teaching about it. I would like to tackle actionable items but so far there is no clear issue described here.

Reply:
Let's go back to history to seek the "Why?".
It seems that the `/es` version started in https://github.com/mui-org/material-ui/pull/8772 for tree shaking support. You solved this problem.
In #5952, it's about accessing the sources to do the transpilation in userland. We can continue in this direction. 

> I'm clearly improving the current situation for the /es build since

How is this better?

Reply:
Let's review the options we have:
- We publish the true sources, without any transpiration steps.
- We publish the source by cleaninh them and transpiling any feature that is not yet into the JavaScript specification (the current approach).
- We transpile the source to support the last version of all the platforms we target. 
- We transpile the source to support x% of the web browsers, x behing a low figure.

Reply:
>> I'm clearly improving the current situation for the /es build since

> How is this better?

The documentation is flat out wrong. We can go back and discuss what the goal is. I don't have the energy to work on bug fixes that are later declared as working as intended. That's just not how things should work.

Reply:
I wasn't raising an objection with the changes. I wanted to get a deeper level though on why we should pick one option vs the other. Don't be surprised if I take the opposite side of your proposal even if I agree with you.

Reply:
> The documentation is flat out wrong.

I don't understand this point. Why is it wrong? In https://material-ui.com/guides/minimizing-bundle-size/#ecmascript we say that it can be used **to** target evergreen browser, it doesn't mean it can be used directly.

Speaking of wrong documentation, I think that updating https://material-ui.com/guides/minimizing-bundle-size/#how-to-reduce-the-bundle-size would be more valuable.

Reply:
> We also publish a second version of the components to target evergreen browsers. 

Interpreting this as "it doesn't target evergreen browsers but you can use it for such a use case" is quite the stretch. I believe that anybody would read this as "this targets evergreen browsers".